src/app/modules/payments/payments.controller.ts

```diff
@@ -66,3 +71,46 @@ export const handleGetPreviousPaymentId: ControllerHandler<
       })
   )
 }
+
+export const handleSendOnboardingEmail: ControllerHandler<
+  unknown,
+  unknown,
+  { email: string }
+> = (req, res) => {
+  const { email } = req.body
+
+  const logMeta = {
+    action: 'handleSendOnboardingEmail',
+    email,
+  }
+
+  return sendOnboardingEmailIfEligible(email)
+    .map(() => res.sendStatus(StatusCodes.OK))
+    .mapErr((error) => {
+      if (error instanceof InvalidDomainError) {
+        logger.info({
+          message: 'Email domain is not whitelisted for onboarding email',
+          meta: logMeta,
+        })
+        // If email domain is not whitelisted, return 403
+        return res.sendStatus(StatusCodes.FORBIDDEN)
```

Comment:
should it be 401 Unauthorized instead? similar to the mapping for existing InvalidDomainError in [auth.utils.ts](https://github.com/opengovsg/FormSG/blob/develop/src/app/modules/auth/auth.utils.ts#L24-L27)

Reply:
Oh hmm but wouldn't 403 be more appropriate? 

Cause 401 is like, you are not authenticated, I don't know you, I can't give you this.

Whilst 403 is like, I know who you are, you are not authorised. (for this case its this right? <-)

https://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses

@LinHuiqing what do you think?

Reply:
ooh gotcha. thanks for that link! 

> 401 Unauthorized, the HTTP status code for authentication errors. And that’s just it: it’s for authentication, not authorization
this makes sense to me!

Reply:
Yea I picked 403 based on what I understood of 401 and 403, which is inline with what @foochifa flagged. And yea I also saw the `InvalidDomainError` flagging 401 instead and decided that they might be wrong based on the same understanding.

Since we're on the topic, I think the `MailSendError` error code shouldn't throw a 400 too (which is current behaviour for our `MailSendError`s), as it should be a server error response instead and has nothing to do with the client. Maybe 502 Bad Gateway or 503 Service Unavailable instead. For this PR I kept it as 400 as I thought it wasn't _that_ important for this to be fixed tho.

Reply:
ok! leave it to you whether to create a new issue or fix those error codes in this PR!

Reply:
Added Linear issue here: FRM-1038